static/css/perdiem.css

```diff
@@ -158,11 +159,26 @@ p.profile-total {
     padding-right: 5%;
 }
 
-#left-align {
-    text-align: left;
+.faq {
```

Comment:
A lot of these have:
```
{
    padding: 2em;
}
```

Perhaps they could share a CSS class with that property, instead of us copying it a bunch of times.